MyCryptoHQ/MyCrypto
# Electron Alpha Prep

### Description

Reworks language, warnings, and some new behavior to make Electron ready for an alpha release.

**Copy is _not_ localized because of its temporary nature.**

### Changes

* Adds a warning each time the user opens the alpha.
* Adds a hard coded expiration date to alpha builds (in `config/data.tsx`) Once this date has passed, the alpha will brick itself.
* Adds an update check done in the webview. Pings github releases, and does a semver check against the current version. If it's lower, pops open a simple modal requesting an update.
* Changes 'BETA RELEASE' banner to say 'ALPHA RELEASE'
* Disables native updater from running at all (for now)

### Steps to Test

1. Run the electron app
2. Confirm you get alpha warning
3. Edit `APP_ALPHA_EXPIRATION` to an earlier date, confirm you get bricked view
4. Edit `VERSION_RC` to a lower version (e.g. 0.9.0), confirm you are prompted to update

## Screenshots

### Alpha Notice

![screen shot 2018-04-24 at 6 11 15 pm](https://user-images.githubusercontent.com/649992/39216744-e3694d76-47ea-11e8-9622-acd82045caff.png)


### Bricked App

![screen shot 2018-04-24 at 5 32 27 pm](https://user-images.githubusercontent.com/649992/39216746-e5853a2a-47ea-11e8-85f6-0f352bf27617.png)

### Update Modal

![screen shot 2018-04-24 at 4 40 26 pm](https://user-images.githubusercontent.com/649992/39216749-ea806d9c-47ea-11e8-9405-58be4ea741ae.png)
